v2/book/examples/SatorImaging.TDoubles.html

```
@@ -0,0 +1,69 @@
+
+<h1>RSCG nr 286 :  SatorImaging.TDoubles</h1>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)